src/library/Uno.Themes.WinUI.Markup/Theme/ProgressBar.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Media;
+using Uno.Extensions.Markup.Internals;
+
+namespace Uno.Themes.Markup
+{
+	public static partial class Theme
+	{
+		public static class ProgressBar
```

Comment:
Missing **MinWidth** and **Height** to match XAML latest changes
https://github.com/unoplatform/Uno.Themes/blob/8c1f8b5c7958430b7fd785b4d9857ea4934a3270/src/library/Uno.Material/Styles/Controls/v2/ProgressBar.xaml#L12

Reply:
@agneszitte where should this go in the class structure? Theme.ProgressBar.Resources.Default.**Size** ? And then have two properties MinWidth and Height

Reply:
included